### ISSUE TITLE
Make long line breaking configurable

### ISSUE THREAD

**Issue by t-b** [universal-ctags/ctags#1015]
Since 7b47ff23 (main: omit patterns for long input lines, 2015-10-15) long tag lines are cut off.
I'm using ctags as something parser-like for C-style function declarations and generate code from the tag lines. The above mentioned change broke that as now the function signatures in the tag file are not complete anymore.

Something like `--pattern-length-limit=0` to turn that feature off would be nice.

If I get some pointers where to add stuff, I can have  a try myself.

**Comment by masatake** [comment]
How about this one?
```
diff --git a/main/entry.c b/main/entry.c
index 25490b3..24de761 100644
--- a/main/entry.c
+++ b/main/entry.c
@@ -676,7 +676,7 @@ static size_t appendInputLine (int putc_func (char , void *), const char *const
 		if (c == CRETURN  ||  c == NEWLINE)
 			break;
 
-		if (length >= Option.patternLengthLimit)
+		if (Option.patternLengthLimit != 0 && length >= Option.patternLengthLimit)
 		{
 			*omitted = TRUE;
 			break;

```

**Comment by t-b** [comment]
@masatake But the option `--pattern-length-limit` is not read, or?
I've created PR #1018 which tries to add that option.

**Comment by masatake** [comment]
> @masatake But the option --pattern-length-limit is not read, or?

Oh!, sorry. I forgot that I just made a stub. I will take a look at the pull request. Thank you.

**Comment by t-b** [comment]
Fixed per #1018 .
